redbot/core/core_commands.py

```diff
@@ -4075,17 +4060,16 @@ async def command_default_disable_cog(self, ctx: commands.Context, *, cogname: s
         **Arguments:**
             - `<cogname>` - The name of the cog to make disabled by default. Must be title-case.
         """
-        cog = self.bot.get_cog(cogname)
-        if not cog:
-            return await ctx.send(_("Cog with the given name doesn't exist."))
+        cog = cogname
+        cogname = cog.qualified_name
         if isinstance(cog, commands.commands._RuleDropper):
             return await ctx.send(_("You can't disable this cog by default."))
         await self.bot._disabled_cog_cache.default_disable(cogname)
         await ctx.send(_("{cogname} has been set as disabled by default.").format(cogname=cogname))
 
     @checks.is_owner()
     @command_manager.command(name="defaultenablecog")
-    async def command_default_enable_cog(self, ctx: commands.Context, *, cogname: str):
+    async def command_default_enable_cog(self, ctx: commands.Context, *, cogname: CogConverter):
```

Comment:
As with command_name->command, this should also be renamed from cogname to cog:
```suggestion
    async def command_default_enable_cog(self, ctx: commands.Context, *, cog: CogConverter):
```
You will need to update the docstring and docs as well.